lua/smear_cursor/draw.lua

```diff
@@ -14,18 +14,67 @@ local MATRIX_CHARACTERS = { "▘", "▝", "▀", "▖", "▌", "▞", "▛", "
 -- Create a namespace for the extmarks
 local cursor_namespace = vim.api.nvim_create_namespace("smear_cursor")
 
-M.initialize_lists = function()
-	if _G.smear_cursor.window_ids == nil then
-		_G.smear_cursor.window_ids = { {} }
-		_G.smear_cursor.n_active_windows = { 0 }
+---@type table<number, {win:number, buf:number, used:boolean}[]>
+M.wins = {}
```

Comment:
Could be in `_G.smear_cursor.wins`, in case the plugin gets unloaded and reloaded? (I don't know if this can happen)

Reply:
That should not be needed...